axiom/nr_txn.c

```diff
@@ -531,6 +532,8 @@ nrtxn_t* nr_txn_begin(nrapp_t* app,
   nt->intrinsics = nro_new_hash();
 
   nt->custom_events = nr_analytics_events_create(app->limits.custom_events);
+  nt->log_events
```

Comment:
This needs to support creating a pool size of 0 - currently `nr_analytics_events_create()` returns a NULL ptr not a pool of size 0.

Reply:
Addressed in 2f54982c76d39280b5c946062eb050dd25e12f8f